aws-kendra-datasource/aws-kendra-datasource.json

```diff
@@ -1201,7 +1201,7 @@
     },
     "WebCrawlerSeedUrl": {
       "type": "string",
-      "pattern": "^(https?):\/\/([^\\s]*)",
+      "pattern": "^(https?)://([^\\s]*)",
```

Comment:
Could this change lead to backward incompatible?

Reply:
These are the same things. The `/` is not a special character, so you don't need to escape it.

Reply:
Alright, to be safe, removed this lint rule and reverted the change.